orchestrator/gravity_utils/src/types/batches.rs

```diff
@@ -17,7 +17,7 @@ pub struct BatchTransaction {
     /// This transactions Cosmos pool id
     pub id: u64,
     /// The senders Cosmos address
```

Comment:
TODO: This needs to be reverted in favor of updating deep_space to include > 32 length address bytes

Reply:
looks like a few things here need deep space upstreaming.